cfpb/cfgov-refresh
# Upgrading Webpack to v4.5, enable caching via the uglifyOptions, and modifying output

Upgrading Webpack to v4.5, enable caching via the uglifyJS Plugin, and modifying Webpack output

## Changes

- Upgraded Webpack to version 4.5
- Modified various Webpack configs to enable caching UglifyJS files and modify console output

## Testing

1. Run gulp build and verify that the output no longer contains entrypoints.
    Example: `Entrypoint Expandable.js = Expandable.js`
2. Run `open node_modules/.cache/uglifyjs-webpack-plugin` and verify that the folder exists.

## Notes

-  We have a couple layers of caching / process avoidance going on, so we will need to pay attention to ensure the caches are being expired properly. 

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
